src/mpid/ch4/src/ch4_types.h

```diff
@@ -317,6 +331,8 @@ typedef struct MPIDI_CH4_Global_t {
     OPA_int_t progress_count;
 
     MPID_Thread_mutex_t vci_lock;
+    int max_runtime_vcis;
+    MPIDI_vci_t vci[MPIDI_CH4_MAX_VCIS];
```

Comment:
Where are these VCIs initialized and finalized? In particular, the mutexes need initializations and finalizations.

Reply:
Correct. Just updated the PR.